app/src/main/java/com/github/orkest/ui/sharing/PlaylistActivity.kt

```diff
@@ -50,34 +61,21 @@ class PlaylistActivity() : ComponentActivity() {
 
         playlistViewModel = PlaylistViewModel(appDao)
 
+        val song = Song()
+
         // check if extras are not null
         Log.d("PlaylistActivity", "songId: ${intent.getStringExtra("songId")}")
-        intent.getStringExtra("songName")?.let {
-            // update database
-            val song = Song(
-                Title = it,
-                Artist = "Unknown Artist",
-                Album = "Unknown Album",
-                URL = "Unknown ID")
-
-            // store song in a way that both users can see them in their pages
-            playlistViewModel.storeSong(
-                song,
-                intent.getStringExtra("senderUsername") ?: "sender1",
-                intent.getStringExtra("receiverUsername") ?: "sender1",
-                this
-            )
-            /*
-            playlistViewModel.storeSong(
-                song,
-                intent.getStringExtra("receiverUsername") ?: "xx",
-                intent.getStringExtra("senderUsername") ?: "xx",
-                this
-            )
+        intent.getStringExtra("songName")?.let { song.Title = it }
+        intent.getStringExtra("songArtist")?.let { song.Artist = it }
```

Comment:
Again nice fix, it should avoid to have Unkwon artist Unknown Album :100:

Reply:
Yess! Now our app is more professional😍